### PR TITLE
Add support for node 8

### PR DESCRIPTION
**Pull request by Wardormeur**
Upgrade of pg-dependants libs is due to a bug in node-pg for node > 6
This triggers a couple of changes (seneca-postgres) which triggers others changes ( conf for camelCase, path to sanitizer..)
Lodash's snakeCase/camelCase is not used becasue it removes the '$' char, which is used in seneca for special columns

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mhart/alpine-node:0.10.48
+FROM node:8-alpine
```
Comment (butlerx) [inline Dockerfile]: would rather use carbon-alpine but doesn't really matter

File: config/config.js [hunk 2]
```diff
@@ -9,7 +11,13 @@ module.exports = function (options) {
       port: process.env.POSTGRES_PORT || 5432,
       username: process.env.POSTGRES_USERNAME,
       password: process.env.POSTGRES_PASSWORD,
-      nolimit: true
+      nolimit: true,
+      fromColumnName: function (attr) {
```
Comment (butlerx) [inline config/config.js]: arrow?

File: config/config.js [hunk 3]
```diff
@@ -9,7 +11,13 @@ module.exports = function (options) {
       port: process.env.POSTGRES_PORT || 5432,
       username: process.env.POSTGRES_USERNAME,
       password: process.env.POSTGRES_PASSWORD,
-      nolimit: true
+      nolimit: true,
+      fromColumnName: (attr) => {
+        return camelCase(attr);
```
Comment (butlerx) [inline config/config.js]: i feel you know this isnt what i meant 😜

Comment (Wardormeur) [reply config/config.js]: ah, fair, can still reduce it
Will do that tomorrow

**Pull request merged**